src/block/Hopper.php

```diff
@@ -57,6 +57,17 @@ public function getStateBitmask() : int{
 		return 0b1111;
 	}
 
+	public function getFacing() : int{ return $this->facing; }
```

Comment:
```suggestion
	public function getFacing() : int{
		return $this->facing;
	}
```

Reply:
This isn't necessary. It's consistent with other implementations.